packages/core/src/admin-ui/components/Navigation.tsx

```diff
@@ -76,7 +76,7 @@ const AuthenticatedItemDialog = ({ item }: { item: AuthenticatedItem | undefined
     >
       {item && item.state === 'authenticated' ? (
         <div css={{ fontSize: typography.fontSize.small }}>
-          Signed in as <strong>{item.label}</strong>
+          Signed in as <strong css={{ display: 'block' }}>{item.label}</strong>
```

Comment:
This is how the `Signed in as` looks like
<img width="572" alt="Screenshot 2022-05-23 at 12 46 50 pm" src="https://user-images.githubusercontent.com/28669082/169733845-5802f9fb-e7e7-4eaf-8db6-b146a6405846.png">